### PR TITLE
Release notes 2021-02 

### PR DESCRIPTION
**Pull request by enikonemeth**

### CODE REVIEW

File: _updates/2021-02-22-february.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+Every sight and every sound.. 🎶
+
+
+Another Valentine's day passed by, and we are here with the news. 
+Check out what we done during these loving days. 
+
+
+### admin-ui@1.17.0
+
+We completely renewed our search menu. Now we have advanced options for searching to make this feature easy to use. 💬 [#1180](https://github.com/SenseNet/sn-client/pull/1180)
```
Comment (herflis) [inline _updates/2021-02-22-february.md]:
```suggestion
We completely renewed the search page. You have now advanced options for searching and filtering  the results to make this feature easy to use. 💬 [#1180](https://github.com/SenseNet/sn-client/pull/1180)
```

File: _updates/2021-02-22-february.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+New options:  ⚡
+- search by type (document, image, event, user, article, workspace, folder, task, memo, group)
+- search in dedicated container
```
Comment (herflis) [inline _updates/2021-02-22-february.md]:
```suggestion
- search in a specific container
```

File: _updates/2021-02-22-february.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+- search for the contents what belongs to me
+- search by creation or modification date
```
Comment (taki9) [inline _updates/2021-02-22-february.md]: filter instead of search?

File: _updates/2021-02-22-february.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+
+### admin-ui@1.17.0
+
+We completely renewed our search menu. Now we have advanced options for searching to make this feature easy to use. 💬 [#1180](https://github.com/SenseNet/sn-client/pull/1180)
+
+New options:  ⚡
+- search by type (document, image, event, user, article, workspace, folder, task, memo, group)
+- search in dedicated container
+- search for the contents what belongs to me
```
Comment (herflis) [inline _updates/2021-02-22-february.md]:
```suggestion
- search for the contents what belongs to me (shared with me, assigned to me, etc.)
```

File: _updates/2021-02-22-february.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+- search by creation or modification date
+
+Note: This search input does not support query syntax, for queries use the command palette instead! 🌀
```
Comment (herflis) [inline _updates/2021-02-22-february.md]:
```suggestion
Note: The advanced search input does not support query syntax, for working with queries use the command palette instead! 🌀
```

File: _updates/2021-02-22-february.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+Saving queries does not change, you can still store your search filters. 💾
```
Comment (herflis) [inline _updates/2021-02-22-february.md]:
```suggestion
The save query feature did not change, you can still store your search filters. 💾
```

File: _updates/2021-02-22-february.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+![Advanced search](/img/advanced_search.gif "Advanced search")
+
+### @@sensenet/default-content-types@3.2.0
+
+Default content types has been extended with some types 👽 [#1163](https://github.com/SenseNet/sn-client/pull/1163)
+
+### @@sensenet/authentication-oidc-react@2.1.0
+
+We had to prepare in case the identity server shuts down 😎  [#1187](https://github.com/SenseNet/sn-client/pull/1187)
```
Comment (herflis) [inline _updates/2021-02-22-february.md]:
```suggestion
We had prepare admin-ui to cases when the identity server is down 😎  [#1187](https://github.com/SenseNet/sn-client/pull/1187)
```

File: _updates/2021-02-22-february.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+
+### @@sensenet/query@1.2.0
+
+Due to advanced search we also has some additions here 😵 [1180](https://github.com/SenseNet/sn-client/pull/1180)
```
Comment (herflis) [inline _updates/2021-02-22-february.md]:
```suggestion
Due to the advanced search feature we also add some changes here 😵 [1180](https://github.com/SenseNet/sn-client/pull/1180)
```

**Review by herflis — CHANGES_REQUESTED** [review]

**Pull request merged**